features/company_details.feature

```diff
@@ -1,52 +1,16 @@
 Feature: Company details
 
-  Background: Navigating to your company details page
-    Given I visit your company details page
-
-   Scenario: Displays header
-    Then I should see 'Your details' header
-
-  Scenario: Displays we need these in case we have to contact you copy
-    Then I should see we need these in case we have to contact you copy
-
-  Scenario: Displays warning
-    Then I should see warning
-
-  Scenario: Displays company name
-    Then I should be asked for the company name
-
-  Scenario: Displays company's address
-    Then I should be asked if the company address is correct
-
-  Scenario: Displays name
-    Then I should be asked your first name
-    And I should be asked your last name
-
-  Scenario: Displays position in company
-    Then I should be asked my position in the company
-    And I should be asked if I am the director
-    And I should be asked if I am the company secretary   
-    And I should be asked if I am the company solicitor
-
-  Scenario: Displays contact number
-    Then I should be asked for a contact number
-
-  Scenario: Displays email address
-    Then I should be asked for my work email address
-
-  Scenario: Successfully submits company details form
-    When I successfully submit the company details
-    And I click on continue
-    Then I am taken to /plea/plea/1
-
-  Scenario: Displays error messages
-    And I click on continue
-    Then I should see error <message> with <link>
-    | message                          | link                        |
-    | Enter the company name           | section_company_name        |
-    | Tell us if the company's address | section_correct_address     |
-    | Enter your first name            | section_first_name          |
-    | Enter your last name             | section_last_name           |
-    | You must tell us your position   | section_position_in_company |
-    | Enter a contact number           | section_contact_number      |
-    
\ No newline at end of file
+  Background:
+    Given a valid company case
+
+  Scenario: Submit details
```

Comment:
We seem to have gone from high level to a low level feature here ?  I thought the this ticket was about using python to do cucumber tests, not actually changing the cucumber tests ?

The original was a very 'non technical' feature - this one has implementation details known such as the field names (i.e. 'first_name', 'last_name').

In my experience with automated browser testing, they should always be written from a users perspective - and users to not know that a field is called 'first_name' etc..

Reply:
This comment pretty much applies to all tests as well

Reply:
Also, this is how ET was to start with and we ended up changing it for consistency with HWF.  This is now inconsistent with hwf and ET.  I feel consistency is important even across language boundaries as QA need to be involved in these tests, as do BA's potentially - we don't want a situation where everyone writes them differently

Reply:
Yeah I agree with this.  We need to agree on a style of writing scenarios and keep it consistent across all apps.